src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,7 +143,18 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * @param courseId
+     * @param userEmail
+     * @return a list of viewable feedback sessions for any user for his course.
+     * @throws EntityDoesNotExistException
```

Comment:
On a separate note, check what happens if the course existence  check is removed altogether but the course does not exist. Perhaps the code deeper below in the call stack throws the EntityDoesNotExistException anyway, in which case we can remove the existence check at this level.